src/cmake-tools.ts

```diff
@@ -1777,6 +1777,15 @@ export class CMakeTools implements vscode.Disposable, api.CMakeToolsAPI {
     if (!dbg && debug_config.type === "cppvsdbg") {
       dbg = "vsdbg";
     }
+    const debug_environment = [];
+    const debug_env_entries = Object.entries(await drv.getCTestCommandEnvironment());
+    for (const [key, value] of debug_env_entries) {
+      debug_environment.push({
+        name: key,
+        value
+      });
+    }
+    debug_config.environment = debug_environment;
```

Comment:
Nit: feels like Object.entries(...).map() is a more concise way?

Reply:
If you insist, otherwise this is more readable:0

Reply:
done